clients/unshaded/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -317,12 +317,9 @@ public synchronized long createFile(TachyonURI path, TachyonURI ufsPath, long bl
     validateUri(path);
     try {
       if (blockSizeBytes > 0) {
-        UnderStorageType underStorageType =
-            mTachyonConf.getEnum(Constants.USER_FILE_UNDER_STORAGE_TYPE_DEFAULT,
-                UnderStorageType.class);
         CreateOptions options =
             new CreateOptions.Builder(ClientContext.getConf()).setBlockSizeBytes(blockSizeBytes)
-                .setRecursive(recursive).setUnderStorageType(underStorageType).build();
```

Comment:
underStorageType for `CreateOptions` is still set, but with default value in construction of `CreateOptions`. this might be a bit confusing, but what we did here essentially is still read the default value of this property and set it in `CreateOptions`---just in a different location.